quiz.py

```diff
@@ -0,0 +1,63 @@
+def startQuiz():
+    while (True):
+        print("Start Quiz")
+        num_questions = howManyQuestions("samplequiz.txt") #count number of quesitons in the file.
+        displayQuestion("samplequiz.txt") #display the question.
+        #would be a list that contains user answers.
+        answers = [input("Your answer for question {}:\n".format(i+1)) for i in range(num_questions)] 
```

Comment:
The user seems to only be able to input numeric answers. Would be better to only allow user to input multiple choice letters (A,B,C,D) or a string for fill-in-the-blank. If the user fails to write a correctly formatted input, the app should print an error message but should allow the user to continue taking the quiz.

Reply:
Actually in this function, the program will expect the user to input multiple choice letters. But I have yet to implement one for fill in the blanks though.

Reply:
I have made a function that will act as an user input verification so that users could only enter 'A', 'B', 'C' or 'D' as an answer.